frontend/src/app/components/about/about.component.scss

```diff
@@ -177,6 +177,10 @@
     }
   }
 
+  .project-member-avatar img {
+    margin: 40px 20px 10px !important;
```

Comment:
use of `!important` is discouraged, can't you fix this in a cleaner way

Reply:
Removed with [6daffe5](https://github.com/mempool/mempool/pull/4939/commits/6daffe5b13ffd59eabc65c8d92ffa38f78b76225).